bin/flip_dns_record

```diff
@@ -10,9 +12,11 @@ if __name__ == "__main__":
                                   aws_access_key_id=os.getenv('AWS_ACCESS_KEY_ID'),
                                   aws_secret_access_key=os.getenv('AWS_SECRET_ACCESS_KEY'),
                                  )
-        current_live = GetCurrentLiveEnvironment(os.getenv('TARGET_DEPLOY_DOMAIN'), aws_client).call()
+        hosted_zone_gateway = HostedZoneGateway(aws_client)
+        current_live = GetCurrentLiveEnvironment(os.getenv('TARGET_DEPLOY_DOMAIN'), hosted_zone_gateway).call()
         print(FlipDNSRecord(
             client=aws_client,
```

Comment:
FlipDNSRecord should be refactored to remove aws client dependency

Reply:
https://github.com/madetech/python-aws-blue-green/issues/10